Tests/DependencyInjection/Fixtures/config/xml/redirect_multi.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<container xmlns="http://symfony.com/schema/dic/services"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:swiftmailer="http://symfony.com/schema/dic/swiftmailer"
+    xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd
+                        http://symfony.com/schema/dic/swiftmailer http://symfony.com/schema/dic/swiftmailer/swiftmailer-1.0.xsd">
+
+    <swiftmailer:config delivery-address="first@host.com"></swiftmailer:config>
+    <swiftmailer:config delivery-address="second@host.com"></swiftmailer:config>
```

Comment:
this is wrong. You should not have multiple config nodes. You should have a single config node containing multiple develivery addresses:

```xml
<swiftmailer:config>
    <swiftmailer:delivery-address>first@host.com</swiftmailer:delivery-address>
    <swiftmailer:delivery-address>second@host.com</swiftmailer:delivery-address>
</swiftmailer:config>
```